src/gui/optionsdialog.ui

```diff
@@ -837,6 +837,9 @@
             </item>
             <item>
              <widget class="QCheckBox" name="checkRecursiveDownload">
+              <property name="toolTip">
+               <string>When a torrent is downloaded, offer to download any .torrent files found inside it</string>
```

Comment:
Sorry, but my previous comment is still relevant:
>It confuses me. I think "download any .torrent files" is not what you really want to do. At least I understand "download .torrent files" literally. Shouldn't it be "add torrents from any .torrent files" instead?

Reply:
@glassez, adem4ik:  I noticed that wording mistake of mine *after* the change had already been committed.  Sorry about that. 

> When a torrent is downloaded, offer to **add torrents from** any .torrent files found inside it